README.md

```diff
@@ -1 +1,62 @@
-# React-Capstone
\ No newline at end of file
+# Capstone Project React - React-Capstone?
+
+> A website that gets the information's (population, name, region, capital and area) about any country in the world, you can search for a country or filter countries by region
+
+### Links
+
+- Solution URL: [GitHub Code](https://github.com/ShaheerCH/React-Capstone)
+- Live Site URL: [React-Capstone](https://mellow-nasturtium-6bd1e4.netlify.app/)
+- Video Presentation: [React-Capstone Video Presentation](https://www.loom.com/share/1e84789da7e64b3482945855c6f54cf8)
+
+## My process
+
+### Built with
+
+- Html
+- Css
+- React
+- Jest
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Have a good knowledge on how to parse UI designs
+- Have VSCode or other text editor installed. [Link to download VSCode](https://code.visualstudio.com/download)
+- Install node package. [Link to download node](https://nodejs.org/en/download/)
+- Have git installed.[Link to download git](https://git-scm.com/downloads)
+- Create a github repository.
+
+## Setup
+
+- `git clone https://github.com/ShaheerCH/React-Capstone.git`
+- Install packages: `npm install`
+- Start Live Server: `npm run dev`
+
+### Continued development/Project Addons
+
+- Create user login and authetication using Forebase
+- Create a protected page for loggen in users
+- Create a working contact form with validation
+
+## Authors
+
+👤 **Shaheer**
+
+- GitHub: [@ShaheerCH](https://github.com/ShaheerCH)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to [Dennis Sakwa](https://www.behance.net/sakwadesignstudio) for the design
```

Comment:
- [ ] According the the project requirements, the person to acknowledge for the design is [Nelson Sakwa](https://www.behance.net/sakwadesignstudio)
![image](https://user-images.githubusercontent.com/52934128/184499129-5e05eda4-b45f-4e0f-9086-b433319c9cb4.png)